### PR TITLE
Insert new committee members

### PR DESCRIPTION
**Pull request by 7coil**

### CODE REVIEW

**Review by djpiper28 — APPROVED** [review]

**Comment by 7coil** [comment]
Whoops!

**Comment by chazzox** [comment]
someone is missing

File: src/posts/people/2021-saurabh/index.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+id: 2021-saurabh
+name: Amber Abuah
```
Comment (djpiper28) [inline src/posts/people/2021-saurabh/index.mdx:3]: might want to change this

Comment (7coil) [reply src/posts/people/2021-saurabh/index.mdx]: Ah, thanks!

**Pull request merged**